tests/unit_tests/test_runner_service.py

```diff
@@ -0,0 +1,252 @@
+from arteria.web.state import State as arteria_state
+from checksum.runner_service import Job, RunnerService
+
+import os
+import tempfile
+import pytest
+import logging
+import asyncio
+
+
+class TestJob:
+    def test_basic_command(self, caplog):
+        """
+        Test basic command is run and logged
+        """
+        job_id = 1
+        cmd = ["echo", "test"]
+        caplog.set_level(logging.INFO)
+
+        job = Job(job_id, cmd)
+        assert job.job_id == job_id
+        assert job.cmd == cmd
+
+        job.wait()
+
+        assert job.get_status() == arteria_state.DONE
+        assert caplog.records[0].levelname == "INFO"
+        assert caplog.records[0].msg == (
+                f"Starting:\n job id: {job_id}\n cmd: {cmd}")
+
+    def test_stdout_to_file(self):
+        """
+        Test it is possible to redirect stdout to a file.
+        """
+        job_id = 2
+        msg = "test"
+        cmd = ["echo", msg]
+        stdout = tempfile.NamedTemporaryFile(mode='r')
+        job = Job(job_id, cmd, stdout=stdout)
+
+        job.wait()
+        stdout.seek(0)
```

Comment:
Hmm, yes, but I'll probably fix the interface so that `Job.__iniit__` takes a string as parameter and takes care of opening/closing the files itself (see my comment earlier).